src/workflow_review.md

```diff
@@ -0,0 +1,128 @@
+- report bugs in packages you're using
+
+Issues can be:
+
+- assigned to someone
+- labeled (e.g. bug, documentation, help wanted,...)
+
+# Add code in a new feature branch
+
+(If you don't have the project locally, check the manual [here](https://inbo.github.io/git-course/course_rstudio.html#23_clone_a_repo_to_work_locally).)
+
+In RStudio on the `Git`tab:
+
+- make sure the `master` branch is active and up to date (`Pull`)
+- add a `New Branch` (hit button) and give it an informative name
+- add new code or change code, and make a `Commit` for every change
+    - you can select specific lines or chunks, you don't have to commit all changes at once
+    - make 1 commit for each change and add a clear commit message
+- regularly `Push` your changes to the repository on github
+
+These steps are described into detail in a [separate workflow](https://inbo.github.io/git-course/workflow_rstudio.html) and in [this manual](https://inbo.github.io/git-course/course_rstudio.html#28_create_logical_commits).
+
+# Create a pull request
+
+In RStudio, make sure all changes are committed and pushed.
+
+In the GitHub repository:
+
+- create a pull request by either
+    - using the `Compare & pull request` button that appears after recent pushes
+    - selecting your new feature branch and hitting the `New pull request` button
+- make sure the correct branches are indicated:
+    - 'base' mentions the branch where will merged to, often the master branch
+    - the 'compare' branch is your feature branch
+- adapt the title if needed, and describe what you did (you can scroll down to check your commits)
+- hit `Create Pull request`
+- request reviewers by adding them on the right top of the page
+
+For an explanation including screenshots, see [here](https://inbo.github.io/git-course/course_rstudio.html#214_pull_requests) or [here](https://inbo.github.io/git-course/workflow_rstudio.html#edits_on_branch_are_ready).
+
+__Remark:__ after creating a pull request, it is still possible to add new
+commits to this branch and pull request.
+After pushing new commits in RStudio, they appear in the pull request.
+To show others that you are not finished yet, you can convert the pull request
+to draft (right top of page, where you request reviewers)
+
+# Revision of code
+
+Reviewers will receive an email with a link to the pull request, and they can:
+
+- inspect
+    - __the changes__ on GitHub in the pull request by inspecting the tabs `Conversation`, `Commits` and `Files changed`
+    - __the adapted version__ locally in RStudio after hitting `Pull` and select the branch name (mentioned at the top of the pull request in GitHub)
+- review it on GitHub by following [this workflow](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review)
+    - when reviewing, do not mix unrelated items in 1 proposed change, to allow the author to selectively accept changes
```

Comment:
But you can't accept all suggestions on a line in the same batch commit.

Reply:
How would you solve this?  Just remove this last line that starts with `when reviewing`?  In fact, while doing, they will probably automatically come a best strategy themselves.

Reply:
solved in 4ecf41d (removed sentence)